src/main/java/com/google/cloud/resourcemanager/OrgPolicyInfo.java

```diff
@@ -0,0 +1,397 @@
+              return orgPolicyInfo.toPb();
+            }
+          };
+  private static final long serialVersionUID = 9148970963697734236L;
+
+  private BooleanPolicy booleanPolicy;
+  private String constraint;
+  private String etag;
+  private ListPolicy listPolicy;
+  private RestoreDefault restoreDefault;
+  private String updateTime;
+  private Integer version;
+
+  /** Used in policy_type to specify how booleanPolicy will behave at this resource. */
+  public static class BooleanPolicy implements Serializable {
+    private static final long serialVersionUID = -2133042982786959351L;
+    private final Boolean enforce;
+
+    BooleanPolicy(Boolean enforce) {
+      this.enforce = enforce;
+    }
+
+    public boolean getEnforce() {
+      return enforce;
+    }
+
+    @Override
+    public String toString() {
+      return MoreObjects.toStringHelper(this).add("enforce", getEnforce()).toString();
+    }
+
+    @Override
+    public int hashCode() {
+      return Objects.hash(getEnforce());
+    }
+
+    @Override
+    public final boolean equals(Object obj) {
+      return obj == this
+          || obj != null
+              && obj.getClass().equals(BooleanPolicy.class)
+              && Objects.equals(toPb(), ((BooleanPolicy) obj).toPb());
+    }
+
+    com.google.api.services.cloudresourcemanager.model.BooleanPolicy toPb() {
+      return new com.google.api.services.cloudresourcemanager.model.BooleanPolicy()
+          .setEnforced(enforce);
+    }
+
+    static BooleanPolicy fromPb(
+        com.google.api.services.cloudresourcemanager.model.BooleanPolicy booleanPolicy) {
+      return new BooleanPolicy(booleanPolicy.getEnforced());
+    }
+  }
+
+  /**
+   * Used in policy_type to specify how listPolicy behaves at this resource.
+   *
+   * <p>ListPolicy can define specific values and subtrees of Cloud Resource Manager resource
+   * hierarchy (Organizations, Folders, Projects) that are allowed or denied by setting the
+   * allowedValues and deniedValues fields. This is achieved by using the under: and optional is:
+   * prefixes. The under: prefix is used to denote resource subtree values. The is: prefix is used
+   * to denote specific values, and is required only if the value contains a ":". Values prefixed
+   * with "is:" are treated the same as values with no prefix. Ancestry subtrees must be in one of
+   * the following formats: - "projects/", e.g. "projects/tokyo-rain-123" - "folders/", e.g.
+   * "folders/1234" - "organizations/", e.g. "organizations/1234" The supportsUnder field of the
+   * associated Constraint defines whether ancestry prefixes can be used. You can set allowedValues
+   * and deniedValues in the same Policy if allValues is ALL_VALUES_UNSPECIFIED. ALLOW or DENY are
+   * used to allow or deny all values. If allValues is set to either ALLOW or DENY, allowedValues
+   * and deniedValues must be unset.
+   */
+  public static class ListPolicy implements Serializable {
```

Comment:
outer class

Reply:
need to kept as it just need removed the public access specifier to be more specific because its only used to define the Type in `OrgPolicyInfo`